.github/workflows/build.yml

```diff
@@ -68,4 +75,46 @@ jobs:
           yarn --immutable
       - name: Build Webchat
         run: |
-          yarn build
+          yarn workspace @botpress/webchat build
+
+  studio:
+    name: Studio
+    runs-on: ubuntu-latest
+    needs: files_changed
+    if: needs.files_changed.outputs.studio == 'true'
+    strategy:
+      matrix:
+        path: ['studio-be', 'studio-ui']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+      - name: Fetch Node Packages
+        run: |
+          yarn --immutable
+      - name: Build Messaging
+        run: |
+          yarn workspace @botpress/${{ matrix.path }} build:tsc
```

Comment:
I think we should have the reflex of type checking all packages that use parcel.